mslib/msui/ui/ui_mainwindow.ui

```diff
@@ -379,9 +379,6 @@ Save a flight track to name it.</string>
    <property name="text">
     <string>&amp;Configuration</string>
    </property>
-   <property name="shortcut">
```

Comment:
removed lines intended?

Reply:
I removed this because this shortcut applies only for mac. Should I include it anyways?